server/routerlicious/packages/routerlicious-base/src/riddler/app.ts

```diff
@@ -4,28 +4,18 @@
  */
 
 import { MongoManager, ISecretManager } from "@fluidframework/server-services-core";
-import { logRequestMetric, Lumberjack } from "@fluidframework/server-services-telemetry";
+import { BaseTelemetryProperties } from "@fluidframework/server-services-telemetry";
 import * as bodyParser from "body-parser";
 import express from "express";
 import morgan from "morgan";
```

Comment:
nit: if we don't want to depend on `morgan` just for types, we can do 1 of 2 things:

1. switch this to `import type { TokenIndexer } from "morgan"` and move morgan to a dev dependency
2. inline the `computeExtraProperties` function in the params to `jsonMorganLoggerMiddleware` which will smartly receive the types from the definition of that function instead of from morgan explicitly

Reply:
Neat! I didn't know that using the inline approach would help with implicitly defining the parameter types. Nice, thanks for the tip!